src/Product/Image/Assignor.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
I feel it's a bit too heavy to be part of the Image object, and also the image needs to be loaded first as well. I find it a bit weird and frustrating that it's the product being assigned to the image and not the image being assigned to the product, but that's the way the cookie crumbles I guess. All this class is doing is loading the file based on its file name and assigning it to a product, it's effectively a way of not letting the controller get much bigger than it already is by shifting the responsibility of moving the image load to a separate class. I  guess I could move it to the `ProductImageCreate` class but then that means it can only be used by the product upload script, I dunno